.github/workflows/ci.yml

```diff
@@ -59,11 +62,14 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: npm
+    - name: Manage Jest cache
+      uses: actions/cache@v3
+      with:
+        path: ${{ matrix.platform.jest-cache }}
+        key: ${{ hashFiles('**/package-lock.json') }}
```

Comment:
We'll start w/ a fresh cache anytime the `package-lock.json` is updated. This seemed like a reasonable balance between having something cached for most runs . . . but not letting it get totally out-of-date.